docs/source/api/fpsemi-examples.rst

```diff
@@ -0,0 +1,499 @@
+     - A presentation for the monoid of orientation reversing mappings.
+
+Full API
+--------
+
+.. py:function:: symmetric_group(n: int, val: author = author.Carmichael) -> List[Tuple[List[int], List[int]]])
+
+   A presentation for the symmetric group.
+  
+   Returns a list giving a monoid presentation for the
+   symmetric group. The argument ``val`` determines the specific presentation
+   which is returned. The options are:
+
+   * ``author.Burnside + author.Miller`` (given on p.464 of `10.1017/CBO9781139237253`_)
+   * ``author.Carmichael`` (given in comment 9.5.2 of `10.1007/978-1-84800-281-4`_)
+   * ``author.Coxeter + author.Moser`` (see Ch. 3, Prop 1.2 of `hdl.handle.net/10023/2821`_)
+   * ``author.Moore`` (given in comment 9.5.3 of `10.1007/978-1-84800-281-4`_)
+
+  
+   The default for ``val`` is ``author.Carmichael``.
+  
+   :param n: the degree
+   :type n: int
+   :param val: the author
+   :type val: author
+  
+   :returns: List[Tuple[List[int], List[int]]]
+
+   :raises RuntimeError: if ``val`` is not listed above (modulo order of author)
+   :raises RuntimeError: if ``n < 4``
+
+   .. _10.1017/CBO9781139237253: https://doi.org/10.1017/CBO9781139237253
+   .. _10.1007/978-1-84800-281-4: https://doi.org/10.1007/978-1-84800-281-4
+   .. _hdl.handle.net/10023/2821: http://hdl.handle.net/10023/2821   
+
+.. py:function:: alternating_group(n: int, val: author) -> List[Tuple[List[int], List[int]]])
+
+   A presentation for the alternating group.
+   
+   Returns a list giving a monoid presentation defining the
+   alternating group of degree ``n``. The argument ``val`` determines the
+   specific presentation which is returned. The options are:
+
+   * ``author.Moore`` (see Ch. 3, Prop 1.3 of `hdl.handle.net/10023/2821`_)
+   
+   The default for ``val`` is ``author.Moore``.
+   
+   :param n: the degree
+   :type n: int
+   :param val: the author
+   :type val: author
+   
+   :returns: List[Tuple[List[int], List[int]]]
+
+   :raises RuntimeError: if ``val`` is not listed above (modulo order of author)
+   :raises RuntimeError: if ``n < 4``
+   
+   .. _hdl.handle.net/10023/2821: http://hdl.handle.net/10023/2821
+
+.. py:function:: full_transformation_monoid(n: int, val: author) -> List[Tuple[List[int], List[int]]])
+
+   A presentation for the full transformation monoid.
+   
+   Returns a list giving a monoid presentation defining the
+   full transformation monoid. The argument ``val`` determines the specific
+   presentation which is returned. The options are:
+
+   * ``author.Aizenstat`` (see Ch. 3, Prop 1.7 of `hdl.handle.net/10023/2821`_)
+   * ``author.Iwahori`` (see Theorem 9.3.1 of `10.1007/978-1-84800-281-4`_)
+   
+   The default for ``val`` is ``author.Iwahori``.
+   
+   :param n: the degree
+   :type n: int
+   :param val: the author
+   :type val: author
+   
+   :returns: List[Tuple[List[int], List[int]]]
+   
+   :raises RuntimeError: if ``val`` is not listed above (modulo order of author)
```

Comment:
```suggestion
   :raises RuntimeError: if ``val`` is not one of the combination of authors listed above (modulo order of author)
```
Basically same comment everywhere in this file please

Reply:
Also it might makes sense to list the exceptions in the same order as the arguments (i.e. swap them here please).